scripts/run_locally.js

```diff
@@ -2,12 +2,13 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+'use strict';
 
-const path = require('path');
-const spawn = require('child_process').spawn;
+var path = require('path');
+var spawn = require('child_process').spawn;
 
 
-const BIN_ROOT = path.join(__dirname, '..', 'server', 'bin');
+var BIN_ROOT = path.join(__dirname, '..', 'server', 'bin');
```

Comment:
Because the linter is ok with the const, but our node interpreter throws an error. Can't figure out why.